velox/functions/sparksql/GetJsonObject.h

```diff
@@ -148,6 +149,11 @@ struct GetJsonObjectFunction {
       // can check the validity of ending character.
       case simdjson::ondemand::json_type::number: {
         switch (rawResult.get_number_type()) {
+          case simdjson::ondemand::number_type::big_integer: {
+            result.append(trimToken(rawResult.raw_json_token()));
```

Comment:
I didn't recognize that we can directly append the result from raw json taken. Could you verify if we can use this for other number types? Looks this way is more efficient.